V2/time_tracker/customers/_domain/_persistence_contracts/_customers_dao.py

```diff
@@ -7,3 +8,19 @@ class CustomersDao(abc.ABC):
     @abc.abstractmethod
     def create(self, data: Customer) -> Customer:
         pass
+
+    @abc.abstractmethod
+    def update(self, id: int, data: Customer) -> Customer:
+        pass
```

Comment:
update should receive the fields to be updated, name, description, in case there are more than 5 fields it should be a dict.

Reply:
This could be talked in the ticket to standardize code